tools/api-markdown-documenter/src/renderers/markdown-renderer/test/RenderSpan.test.ts

```diff
@@ -70,7 +70,7 @@ describe("Span Markdown rendering tests", () => {
 			const node3 = new PlainTextNode(text2);
 			const span = new SpanNode([node1, node2, node3], formatting);
 			expect(testRender(span, { insideTable: true })).to.equal(
-				`<span><b><i>This is some text.</i></b> <br><b><i>${text2}</i></b></span>`,
+				`<span><b><i>This is some text. <br>This is more text!</i></b></span>`,
```

Comment:
Bug fix, it seems?

Reply:
No, just a formatting difference. / making the "expected" string a bit more consistent by not leveraging an existing variable in one place but not the other.

Reply:
This change was actually reverted with my most recent changes. I realized that the way I had implemented styling was incorrect. Many of these such differences are now gone.